kbfsfuse/main.go

```diff
@@ -24,6 +24,7 @@ var runtimeDir = flag.String("runtime-dir", os.Getenv("KEYBASE_RUNTIME_DIR"), "r
 var label = flag.String("label", os.Getenv("KEYBASE_LABEL"), "label to help identify if running as a service")
 var mountType = flag.String("mount-type", defaultMountType, "mount type: default, force, none")
 var version = flag.Bool("version", false, "Print version")
+var servicemount = flag.Bool("service-mount", false, "get mount path from serfice")
```

Comment:
That was my question, if we ever want to change it on other platforms, "drive letter" will be the wrong thing to call it.